servers/Azure.Mcp.Server/docs/azmcp-commands.md

```diff
@@ -1328,7 +1328,7 @@ azmcp get bestpractices get --resource <resource> --action <action>
 azmcp tools list
 
 # List only the available top-level service namespaces
-azmcp tools list --namespaces
+azmcp tools list --namespace-mode
```

Comment:
We should update these docs to include the new options.

Reply:
Updated doc.